src/patternfly/components/Accordion/accordion.scss

```diff
@@ -31,6 +31,8 @@
 
   // accordion toggle icon
   --pf-c-accordion__toggle-icon--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-accordion__toggle-icon--expanded--Transition: .2s ease-in 0s;
```

Comment:
Does anyone recall why these use `expanded` over `m--expanded` `--pf-c-accordion__toggle-text--expanded--Color` `--pf-c-accordion__toggle-text--expanded--FontWeight`

Reply:
Looks like an oversight to me.

Reply:
will create a follow up issue thanks @mcoker

Reply:
will create a follow up issue thanks @mcoker

Reply:
will create a follow up issue thanks @mcoker